dev-system/dev/user.clj

```diff
@@ -29,6 +32,26 @@
           (fn [app-map]
             (u/map-values #(assoc-in % [:log :level] level) app-map))))
 
+(defn set-mode!
+  "Set the mode to use when starting/resetting. Valid options are:
+
+  * :in-memory
+  * :external"
+  [new-mode]
```

Comment:
We can't use one var here for mode. We want to support any combination of in-memory/external for both the database and Elasticsearch, so you need to different vars.